script.js

```diff
@@ -151,62 +154,63 @@ function JSprojectDesktop() {
-                Previous project
-            </div>
-            <div class="direction-right-desk">
-                Next project
-                <img src="images.svg/Union (12).svg" alt="">
-            </div>
-        </div>
-        </div>
-    </div> 
-</div>`);
+          <div class="carousel-desk">
+              <button id="slide-left-desk">
+                  <img src="images.svg/Vector 4.svg" alt="">
+              </button>
+              <ul class="carousel-track-desk">
+                  <li class="carousel-slide-desk"><img src="images.svg/Rectangle 34 (1).svg" class="carousel-images-desk" alt=""></li>
+                  <li class="carousel-slide-desk"><img src="images.svg/Rectangle 34 (1).svg" class="carousel-images-desk" alt=""></li>
+                  <li class="carousel-slide-desk"><img src="images.svg/Rectangle 34 (1).svg" class="carousel-images-desk" alt=""></li>
+                  <li class="carousel-slide-desk"><img src="images.svg/Rectangle 34 (1).svg" class="carousel-images-desk" alt=""></li>
+              </ul>
+              <button id="slide-right-desk">
+                  <img src="images.svg/Vector 5.svg" alt="">
+              </button>
+          </div>
+          <div class="carousel-nav-desk">
+              <img src="images.svg/Rectangle 41 (2).svg" class="carousel-nav-images-desk  carousel-indicator" alt="">
+              <img src="images.svg/Rectangle 27 (2).svg" class="carousel-nav-images-desk" alt="">
+              <img src="images.svg/Rectangle 41 (2).svg" class="carousel-nav-images-desk" alt="">
+              <img src="images.svg/Rectangle 41 (2).svg" class="carousel-nav-images-desk" alt="">
+          </div>
+          <div class="rest-of-the-page-desk">
+              <P id="paragraph-1-desk">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi</P>
+  
+          <p id="paragraph-2-desk"> Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.</P>
+          <div class="pop-up-buttons-desk">
+              <button id="first-button-desk"><a href="">See live</a>  <img src="images.svg/Union (10).svg" alt=""></button>
+              <button><a href="">See source</a>   <img src="images.svg/Vector (31).svg" alt=""></button>
+          </div>
+          <div class="direction-desk">
+              <div class="direction-left-desk">
+                  <img src="images.svg/Union (11).svg" alt="">
+                  Previous project
+              </div>
+              <div class="direction-right-desk">
+                  Next project
+                  <img src="images.svg/Union (12).svg" alt="">
+              </div>
+          </div>
+          </div>
+      </div> 
+  </div>`;
```

Comment:
- [ ] Nice work generating the projects :+1: However it would be nice if you populate the projects from an array object containing all the information about the project.